sonarqube/resource_sonarqube_project_test.go

```diff
@@ -51,8 +51,8 @@ func testAccSonarqubeProjectSettingsConfig(rnd string, name string, project stri
 		  visibility = "%[4]s"
 
 		  setting {
-			key   = "sonar.demo"
-			value = "sonarqube@example.org"
+			key   = "sonar.docker.activate"
```

Comment:
why change these settings in this test - doesn't seem necessary for your PR is it?

Reply:
I just wanted yo use real examples of setting that I can also check via UI that's all ok

Reply:
I am open to change to other configurable settings that are available in all version (community, developer, enterprise)
Any suggestion?